content/en/tracing/services/service_renaming.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
This feature is in preview and requires customer access, what is the current pattern to explicit it?
Would it be something similar to this page
<img width="781" height="214" alt="Screenshot 2025-09-04 at 11 59 06" src="https://github.com/user-attachments/assets/9c8dc4f4-cffb-4d17-935c-427d6caacf82" />